src/App.tsx

```diff
@@ -17,18 +54,34 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                setQuery={setQuery}
+                setFilter={setFilter}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {todos.length < 1 && (
+                <Loader />
+              )}
+              {todos.length > 0 && (
+                <TodoList
```

Comment:
```suggestion
              {!todos.length ? (
                <Loader />
              ) : (
                <TodoList
```
suggest to make ternary here